rpmfusion-free-tainted.repo

```diff
@@ -0,0 +1,33 @@
+[rpmfusion-free-tainted]
+name=RPM Fusion for Fedora $releasever - Free tainted
+#baseurl=http://download1.rpmfusion.org/free/fedora/tainted/$releasever/$basearch/
+metalink=https://mirrors.rpmfusion.org/metalink?repo=free-fedora-tainted-$releasever&arch=$basearch
+enabled=0
+metadata_expire=14d
```

Comment:
This metada_expire is probably right for the free-tainted repo with the libdvdcss in mind.
But it should be more shaped around an updates repo instead
(using metadata_expire=6h)